dev-build/conf/routes

```diff
@@ -395,7 +395,6 @@ GET            /$path<[^/]+(/[^/]+)?>
 
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/gallery/.*>/lightbox.json                                                             controllers.GalleryController.lightboxJson(path)
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/gallery/.*>.json                                                                      controllers.GalleryController.renderJson(path)
-GET            /$path<[\w\d-]*(/[\w\d-]*)?/gallery/.*>/amp                                                                       controllers.GalleryController.render(path)
```

Comment:
This is seems to be the only place this is referenced. It was probably left over from a few months ago when we thought about galleries on amp. That won't happen now for a while.